cypress/integration/auth_spec.js

```diff
@@ -7,25 +7,23 @@ describe('The Auth Page', function () {
             cy.url().should('eq', Cypress.config().baseUrl);
         }); 
     });
-
     describe('auth cookie', function () {
         beforeEach(function() {
             cy.visit('/');
-            const username = 'jane'
-            cy.get('input').type(username)
-            cy.get('button').click() 
+            const username = 'jane';
+            cy.get('input').type(username);
+            cy.get('button').click() ;
         });
-        
         afterEach(() => {
             cy.request({
                 url: '/api/reset-server',
                 method: 'POST'
             });
-            cy.visit('/')
-        })
+            cy.visit('/');
+        });
         it('sends you to dashboard', function() {
             cy.visit('/dashboard');
             cy.url().should('include', '/dashboard');
         });
     });
-});
+});
```

Comment:
You should have an empty line at the end of each file

Reply:
lol peak  ill change that back aha